test-it/testList.json

```diff
@@ -1,10 +1,5 @@
 {
     "tests": {
-        "disabled": {
-            "tests": [
-                "TestVI_VariantEquality_VariantsWithVariantsNestedInsideThem.via"
```

Comment:
I tried to just enable this and see what happens. There was only one bug that I hat to fix to make this work. So, enabled it and wrote a couple of more complex tests and added them to the PR.